doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/con_Troubleshooting.adoc

```diff
@@ -1,11 +1,19 @@
 [id='ref_Troubleshooting']
 = Troubleshooting
 
+[[Retrying_a_Migration_Plan]]
+== Retrying a Failed Migration Plan
+
+If one or more virtual machines in a migration plan fails to migrate, the plan's migration status is `Failed`. You can retry a migration plan to migrate the virtual machines that failed to migrate:
+
+. Click the migration plan to go to the details page to identify the virtual machines that failed to migrate.
```

Comment:
Hi Chris, I don't think that's necessary, because a failed migration plan is right in your face. You run the migration plan. If it succeeds, you get "Success" messages. If it fails, you see a lot of Xs beside the failed VMs or a different message. You don't have to navigate to another page.